articles/server-platforms/symfony/01-login.md

```diff
@@ -11,6 +11,15 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-symfony-php-web-app/tree/master/00-Starter-Seed',
+  pkgRepo: 'auth0-symfony-php-web-app',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: null,
+  pkgType: 'null'
```

Comment:
@mehreencs87 The zip download gives an error `{"status":400,"message":"Invalid type.","error":{"status":400}}` The valid values are: server, js, replace, none (see: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects)

The download URL is you want to check yourself is: https://auth0.com/docs/package/auth0-symfony-php-web-app/master?path=00-Starter-Seed&file_path=&type=null